olive/__init__.py

```diff
@@ -20,6 +20,8 @@
 def set_default_logger_severity(level):
```

Comment:
btw, do we really need both log_severity_level and ort_log_severity_level?

Reply:
we want to default to ERROR for ort but INFO for olive so I think separate levels is helpful. Otherwise, ort makes the logs messy with it's warnings about graphs, etc.